packages/toolkit/tsconfig.base.json

```diff
@@ -32,18 +35,37 @@
     "allowSyntheticDefaultImports": true,
     "emitDeclarationOnly": true,
     "baseUrl": ".",
-    "types": ["vitest/globals"],
+    "types": [
+      "vitest/globals"
+    ],
     "paths": {
-      "@reduxjs/toolkit": ["src/index.ts"], // @remap-prod-remove-line
-      "@reduxjs/toolkit/react": ["src/react/index.ts"], // @remap-prod-remove-line
-      "@reduxjs/toolkit/query": ["src/query/index.ts"], // @remap-prod-remove-line
-      "@reduxjs/toolkit/query/react": ["src/query/react/index.ts"], // @remap-prod-remove-line
+      "@reduxjs/toolkit": [
+        "src/index.ts"
+      ], // @remap-prod-remove-line
+      "@reduxjs/toolkit/react": [
+        "src/react/index.ts"
+      ], // @remap-prod-remove-line
+      "@reduxjs/toolkit/query": [
+        "src/query/index.ts"
+      ], // @remap-prod-remove-line
+      "@reduxjs/toolkit/query/react": [
+        "src/query/react/index.ts"
+      ], // @remap-prod-remove-line
       // for type imports in tests only
-      "@reduxjs/toolkit/dist/*": ["src/*"], // @remap-prod-remove-line
+      "@reduxjs/toolkit/dist/*": [
+        "src/*"
+      ], // @remap-prod-remove-line
       // for type imports in tests only
```

Comment:
These moved comments might prove problematic in some tests

Reply:
prettier moment - will fix